examples/example_server_forum.py

```diff
@@ -0,0 +1,155 @@
+import socket
+import sys
+import socketserver
+from http.server import BaseHTTPRequestHandler
+from http.server import HTTPServer
+import urllib.parse
+
+# using port numbers prepended with 9 to avoid calling sudo during test
+servport = 9090
+posts_file = "./forum/posts.txt"
+
+def jsonify(user, msg):
```

Comment:
Same comments as for client jsonify. If we had more complex logic, I'd recommend refactoring shared parsing functionality out into a separate module, but it's probably overkill in our case.